soothe/test.py

```diff
@@ -147,7 +148,8 @@ def run(self, result: Result) -> None:
                 except (IndexError, ValueError):
                     result.vmaf_result = EncodeTestResult.FAIL
 
-        if not self.params.keep_files \
-           and os.path.exists(output_filepath) \
-           and os.path.isfile(output_filepath):
-            os.remove(output_filepath)
+        if not self.params.keep_files:
+            output_dir = os.path.dirname(output_filepath)
+            print(output_dir)
```

Comment:
is this message a developing left over?

Reply:
what process does generate a directory?

Reply:
yes this is an incorrect solution. See the new patchset